Stubs for GLCanvas missing a few things

<!-- For bugs or other problems please provide the following details in addition to
     your issue report, if applicable. See also https://wxpython.org/pages/how-to-submit-issue/

     For issues about building on Linux, please read this page before reporting it here:
     https://wxpython.org/blog/2017-08-17-builds-for-linux-with-pip/
-->

**Operating system**: linux/macOS
**wxPython version & source**:   4.1
**Python version & source**:      any

**Description of the problem**:

The generated stubs used when wxWidgets was not able to find OpenGL headers appears to be missing a few things:

```
 [675/917] Compiling sip/cpp/sip_html2wxWebViewEvent.cpp
    In file included from ../../../../sip/cpp/sip_glcanvaswxGLAttributes.cpp:10:
    ../../../../sip/cpp/sipAPI_glcanvas.h:682:23: error: aggregate ‘wxGLAttributes _NullGLAttributes’ has incomplete type and cannot be defined
      682 | static wxGLAttributes _NullGLAttributes;
          |                       ^~~~~~~~~~~~~~~~~
    ../../../../sip/cpp/sipAPI_glcanvas.h:683:25: error: aggregate ‘wxGLContextAttrs _NULLGLContextAttrs’ has incomplete type and cannot be defined
      683 | static wxGLContextAttrs _NULLGLContextAttrs;
          |                         ^~~~~~~~~~~~~~~~~~~
    In file included from ../../../../sip/cpp/sip_glcanvaswxGLAttributes.cpp:10:
    ../../../../sip/cpp/sipAPI_glcanvas.h:865:198: error: ‘wxGLCanvasName’ was not declared in this scope; did you mean ‘wxGLCanvas’?
      865 |     wxGLCanvas(wxWindow *parent, const wxGLAttributes &dispAttrs, wxWindowID id=wxID_ANY, const wxPoint &pos=wxDefaultPosition, const wxSize &size=wxDefaultSize, long style=0, const wxString &name=wxGLCanvasName, const wxPalette &palette=wxNullPalette)
          |                                                                                                                                                                                                      ^~~~~~~~~~~~~~
          |                                                                                                                                                                                                      wxGLCanvas
```

See: https://groups.google.com/g/wxpython-users/c/OqEubNllfu4/m/Y_5cQUHHBAAJ

--- a/etg/_glcanvas.py
+++ b/etg/_glcanvas.py
@@ -64,10 +64,9 @@ def run():
     module.addHeaderCode('#include <wx/glcanvas.h>')
 
     tools.generateStubs('wxUSE_GLCANVAS', module,
-                        extraHdrCode=('static wxGLAttributes _NullGLAttributes;\n'
-                                      'static wxGLContextAttrs _NULLGLContextAttrs;\n'),
-                        typeValMap={'wxGLAttributes &': '_NullGLAttributes',
-                                    'wxGLContextAttrs &': '_NULLGLContextAttrs',
+                        extraHdrCode='#define wxGLCanvasName wxT("GLCanvas")\n',
+                        typeValMap={'wxGLAttributes &': '*this',
+                                    'wxGLContextAttrs &': '*this',
                                     })
 
     c = module.find('wxGLContext')

